### ISSUE TITLE
java.lang.NullPointerException when rendition:layout meta is present but empty in v4.0.2

### ISSUE THREAD

**Issue by pettarin** [IDPF/epubcheck#727]
# Environment

EpubCheck v4.0.2 (+ java version "1.8.0_45", Debian testing, but I think the Java version and the OS are irrelevant)

# Symptoms

(Note: the EPUB was patched as described below.)

```
$ epubcheck epub30-test-0100-20160116.epub
Validating using EPUB version 3.0.1 rules.
ERROR(RSC-005): epub30-test-0100-20160116.epub/EPUB/package.opf(33,45): Error while parsing file 'character content of element "meta" invalid; must be a string with length at least 1 (actual length was 0)'.
ERROR(RSC-005): epub30-test-0100-20160116.epub/EPUB/package.opf(33,45): Error while parsing file 'The value of the 'rendition:layout' property must be either 'reflowable' or 'pre-paginated''.
...
java.lang.NullPointerException
        at com.adobe.epubcheck.ctc.EpubSpineCheck.tooManySpineElements(EpubSpineCheck.java:77)
        at com.adobe.epubcheck.ctc.EpubSpineCheck.validate(EpubSpineCheck.java:36)
        at com.adobe.epubcheck.ctc.CheckManager.checkPackage(CheckManager.java:58)
        at com.adobe.epubcheck.api.EpubCheck.doValidate(EpubCheck.java:219)
        at com.adobe.epubcheck.tool.EpubChecker.validateFile(EpubChecker.java:161)
        at com.adobe.epubcheck.tool.EpubChecker.processFile(EpubChecker.java:495)
        at com.adobe.epubcheck.tool.EpubChecker.run(EpubChecker.java:255)
        at com.adobe.epubcheck.tool.Checker.main(Checker.java:31)
```

# Steps to reproduce

Just add  `<meta property="rendition:layout" />` to the `<metadata>` OPF of an otherwise valid EPUB. For example, just add that line to the `epub30-test-0100-20160116.epub` from the EPUB Test Suite, as in attachment: [epub30-test-0100-20160116.epub.zip](https://github.com/IDPF/epubcheck/files/668513/epub30-test-0100-20160116.epub.zip)

# Culprit

https://github.com/IDPF/epubcheck/blob/master/src/main/java/com/adobe/epubcheck/ctc/EpubSpineCheck.java#L77

I guess ``e.getValue()`` returns ``null``. A guard should be put in place to avoid the null pointer exception.

# Notes
Note that this is a regression from v3.0.1, which reported the errors, but it did not throw the null pointer exception.

**Comment by tofi86** [comment]
Thanks for reporting this, Alberto! I got this fixed with 42d7529 and added a test case.

**Comment by pettarin** [comment]
Excellent, thank you.

One day I will tell you the story of how I came across this issue.

I forgot to add that I have not investigated whether similar issues affect other metas whose value is inspected because of specific requirements of the spec (e.g. dcterms:modified?).
